src/tools/cephfs/top/setup.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+from setuptools import setup
+
+__version__ = '0.0.1'
+
+setup(
+    name='cephfs-top',
+    version=__version__,
+    description='top(1) like utility for Ceph Filesystem',
+    keywords='cephfs, top',
+    scripts=['cephfs-top'],
+    install_requires=[
+        'rados',
+        'curses',
```

Comment:
i don't understand why `curses` is listed here. as it is part of the python standard library.

Reply:
right -- I realized when making changed to spec file, but forgot to remote from here.